Client.xcodeproj/project.pbxproj

```diff
@@ -6374,7 +6375,8 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				INFOPLIST_FILE = XCUITests/Info.plist;
-				PROVISIONING_PROFILE_SPECIFIER = FirefoxXCUITests;
+				PRODUCT_BUNDLE_IDENTIFIER = "";
+				PROVISIONING_PROFILE_SPECIFIER = XCUITests;
```

Comment:
I think these needs to be `Bitrise Firefox iOS Dev - XCUI Tests` because that is the name of the provisioning profile in Apple dev site.